cross compilation issue

Hey @ccouzens !

Thanks for your tesseract crates. I'm writing an app that reads texts from cards from an older game I have lying around. I want to run my code on a Raspberry Pi and need to cross-compile your crates for `aarch64-unknown-linux-gnu`.

## Setup 

1. clone `https://github.com/ccouzens/tesseract-plumbing` 
2. `cargo install cross` 
3. add `Cross.toml`  with this content:

```
[target.aarch64-unknown-linux-gnu]
image = "ghcr.io/cross-rs/aarch64-unknown-linux-gnu:main"
pre-build = [
    "dpkg --add-architecture arm64 ",
    "apt-get update ",
    "DEBIAN_FRONTEND=noninteractive apt-get install -y --no-install-recommends apt-utils libc6-dev:arm64 libleptonica-dev libtesseract-dev clang",
]
```
4. run `cross build --release --target=aarch64-unknown-linux-gnu` 

## Error

```
debconf: delaying package configuration, since apt-utils is not installed
   Compiling futures-util v0.3.28
   Compiling clap_derive v4.3.0
   Compiling async-reactor-trait v1.1.0
   Compiling pinky-swear v6.1.0
   Compiling tesseract-plumbing v0.10.0
error[E0432]: unresolved import `self::tesseract_sys::TessBaseAPIInit5`
 --> /cargo/registry/src/github.com-1ecc6299db9ec823/tesseract-plumbing-0.10.0/src/tess_base_api.rs:9:5
  |
9 |     TessBaseAPIInit5, TessBaseAPIMeanTextConf, TessBaseAPIRecognize, TessBaseAPISetImage,
  |     ^^^^^^^^^^^^^^^^
  |     |
  |     no `TessBaseAPIInit5` in the root
  |     help: a similar name exists in the module: `TessBaseAPIInit1`

For more information about this error, try `rustc --explain E0432`.
error: could not compile `tesseract-plumbing` due to previous error
warning: build failed, waiting for other jobs to finish...
```

This occurs because for some reason the generated `capi_bindings.rs` only contains the methods up to `TessBaseAPIInit4` - `TessBaseAPIInit5` is missing.

## More maybe helpful details?

- The image `ghcr.io/cross-rs/aarch64-unknown-linux-gnu:main` is based on 20.04.6 LTS (Focal Fossa)
- The method in question is indeed missing in that tesseract version (4.1.1-2build2 is installed on Focal):

```
root@d30866a02c60:/tesseract-plumbing# grep -i "TessBaseAPIInit" /usr/include/tesseract/capi.h
TESS_API int TESS_CALL TessBaseAPIInit(
TESS_API int TESS_CALL TessBaseAPIInit1(TessBaseAPI* handle,
TESS_API int TESS_CALL TessBaseAPIInit2(TessBaseAPI* handle,
TESS_API int TESS_CALL TessBaseAPIInit3(TessBaseAPI* handle,
TESS_API int TESS_CALL TessBaseAPIInit4(
TESS_API int TESS_CALL TessBaseAPIInitLangMod(TessBaseAPI* handle,
TESS_API void TESS_CALL TessBaseAPIInitForAnalysePage(TessBaseAPI* handle);
TESS_API void TESS_CALL TessBaseAPIInitTruthCallback(TessBaseAPI* handle,
root@d30866a02c60:/tesseract-plumbing# 
```

Any thoughts?

--- a/Cargo.toml
+++ b/Cargo.toml
@@ -18,6 +18,5 @@ thiserror = "1.0"
 image = "0.23.14"
 
 [features]
-default = ["full"]
-full = ["tesseract_5_2"]
+default = ["tesseract_5_2"]
 tesseract_5_2 = []

--- a/Cargo.toml
+++ b/Cargo.toml
@@ -16,3 +16,8 @@ thiserror = "1.0"
 
 [dev-dependencies]
 image = "0.23.14"
+
+[features]
+default = ["full"]
+full = ["tesseract_5_2"]
+tesseract_5_2 = []

--- a/README.md
+++ b/README.md
@@ -9,6 +9,10 @@ This is designed to expose the C API of
 Adding value by deviating from the API is a non-goal. That is left to libraries
 that build on top of `tesseract-plumbing`.
 
+## Requirements/Features
+
+Requires Tesseract version `5.2.0` or newer by default. Use `--no-default-features` if using an older version.
+
 ## Motivation
 
 I'm a maintainer of both [leptess](https://crates.io/crates/leptess) and

--- a/src/tess_base_api.rs
+++ b/src/tess_base_api.rs
@@ -1,16 +1,19 @@
 extern crate tesseract_sys;
 extern crate thiserror;
 
+#[cfg(feature = "tesseract_5_2")]
+use self::tesseract_sys::TessBaseAPIInit5;
 use self::tesseract_sys::{
     TessBaseAPIAllWordConfidences, TessBaseAPICreate, TessBaseAPIDelete, TessBaseAPIGetAltoText,
     TessBaseAPIGetComponentImages, TessBaseAPIGetHOCRText, TessBaseAPIGetInputImage,
     TessBaseAPIGetLSTMBoxText, TessBaseAPIGetSourceYResolution, TessBaseAPIGetTsvText,
     TessBaseAPIGetUTF8Text, TessBaseAPIGetWordStrBoxText, TessBaseAPIInit2, TessBaseAPIInit3,
-    TessBaseAPIInit5, TessBaseAPIMeanTextConf, TessBaseAPIRecognize, TessBaseAPISetImage,
-    TessBaseAPISetImage2, TessBaseAPISetPageSegMode, TessBaseAPISetRectangle,
-    TessBaseAPISetSourceResolution, TessBaseAPISetVariable, TessDeleteIntArray, TessOcrEngineMode,
-    TessPageIteratorLevel, TessPageSegMode,
+    TessBaseAPIMeanTextConf, TessBaseAPIRecognize, TessBaseAPISetImage, TessBaseAPISetImage2,
+    TessBaseAPISetPageSegMode, TessBaseAPISetRectangle, TessBaseAPISetSourceResolution,
+    TessBaseAPISetVariable, TessDeleteIntArray, TessOcrEngineMode, TessPageIteratorLevel,
+    TessPageSegMode,
 };
+
 use self::thiserror::Error;
 use crate::Text;
 use leptonica_plumbing::Pix;
@@ -132,6 +135,7 @@ impl TessBaseApi {
         Self(unsafe { TessBaseAPICreate() })
     }
 
+    #[cfg(feature = "tesseract_5_2")]
     /// Wrapper for [`Init-1`]https://tesseract-ocr.github.io/tessapi/5.x/a02438.html#a2be07b4c9449b8cfc43e9c26ee623050
     pub fn init_1(
         &mut self,

